docs/videos/running_all_devs.md

```diff
@@ -0,0 +1,41 @@
+# Running the All Core Devs call
+
+Create draft agenda file titled “alldevs[date].md” with the date of the next call in YYYYMMDD format.
```

Comment:
I would just leave this up as a pull request that we update as we go based on proposed agenda items, and then merge this PR after the call is complete with all the relevant details.

Reply:
@sohkai  I want the file merged so it can be linked in the Overview page per the last bulletpoint in this section.

Reply:
@sohkai I'm just going to say do whatever works for you. Don't let me micromanage how you run your call. Just consider this PR scaffolding around which you can build your actual process. Feel free to modify to suit your needs.